packages/_vue3-migration-test/src/x-modules/facets/components/test-filters-injection.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+  import FiltersSearch from '../../../../../x-components/src/x-modules/facets/components/lists/filters-search.vue';
+  import SlicedFilters from '../../../../../x-components/src/x-modules/facets/components/lists/sliced-filters.vue';
+  import SortedFilters from '../../../../../x-components/src/x-modules/facets/components/lists/sorted-filters.vue';
+  import ExcludeFiltersWithNoResults from '../../../../../x-components/src/x-modules/facets/components/lists/exclude-filters-with-no-results.vue';
+  import FiltersList from '../../../../../x-components/src/x-modules/facets/components/lists/filters-list.vue';
+  import { useState } from '../../../../../x-components/src/composables/use-state';
+  import { useRegisterXModule } from '../../../../../x-components/src/composables/use-register-x-module';
+  import { facetsXModule } from '../../../../../x-components/src/x-modules/facets/x-module';
+
+  useRegisterXModule(facetsXModule);
```

Comment:
I am not sure why is needed to register the module here. It already has to be registered by component themselves